WcaOnRails/db/structure.sql

```diff
@@ -1341,6 +1341,7 @@ CREATE TABLE `wcif_extensions` (
 /*!50001 SET character_set_results     = utf8mb4 */;
 /*!50001 SET collation_connection      = utf8mb4_unicode_ci */;
 /*!50001 CREATE ALGORITHM=UNDEFINED */
+
```

Comment:
Interestingly this doesn't seem to happen for me:

![image](https://user-images.githubusercontent.com/17034772/62429851-8a5b9f00-b714-11e9-9c4d-916b8ac2c2a9.png)

Reply:
Are you asking why there's a blank space here on line 1344? That's because with these changes, we're now `gsub`-ing away the `DEFINER=` line, which leaves behind a blank line. It might be possible to get rid of the blank line (doing a multiline regex? something else?), but it seemed like it would be extra work with no real benefit.

Reply:
Ah gotcha! I believe it should be as simple as specifying `\n` instead of line start `^`. I'm happy with either version, just wanted to ensure that I won't run into other weird diff locally =)

Reply:
Oh dang, I wasn't expecting that to work, I thought I'd need to enable multiline regexes somehow...

Changed! Now there is no diff to `structure.sql` =)

Reply:
I think the only difference with multiline regexps is that `.` additionally match newline characters, but you can always use them explicitly =)